lib/constants.js

```diff
@@ -11,16 +11,26 @@ governing permissions and limitations under the License.
 */
 
 const { PROD_ENV, STAGE_ENV } = require('@adobe/aio-lib-env')
+const { isInternalToAdobeRuntime } = require('./utils')
 
 // gets these values if the keys are set in the environment, if not it will use the defaults set
+// omit the protocol (https)
 const {
-  ADOBE_STATE_STORE_ENDPOINT_PROD = 'https://storage-state-amer.app-builder.adp.adobe.io',
-  ADOBE_STATE_STORE_ENDPOINT_STAGE = 'http://storage-state-amer.stg.app-builder.corp.adp.adobe.io'
+  ADOBE_STATE_STORE_ENDPOINT_PROD = 'storage-state-amer.app-builder.adp.adobe.io',
+  ADOBE_STATE_STORE_ENDPOINT_STAGE = 'storage-state-amer.stg.app-builder.corp.adp.adobe.io',
+  ADOBE_STATE_STORE_ENDPOINT_PROD_INTERNAL = 'storage-state-amer.app-builder.int.adp.adobe.io',
+  ADOBE_STATE_STORE_ENDPOINT_STAGE_INTERNAL = 'storage-state-amer.stg.app-builder.int.adp.adobe.io',
+  API_VERSION = 'v1beta1',
+  ADOBE_STATE_STORE_REGIONS = [ // first region is the default region
+    'amer',
+    'apac',
+    'emea'
+  ]
 } = process.env
 
 const ADOBE_STATE_STORE_ENDPOINT = {
-  [PROD_ENV]: ADOBE_STATE_STORE_ENDPOINT_PROD,
-  [STAGE_ENV]: ADOBE_STATE_STORE_ENDPOINT_STAGE
+  [PROD_ENV]: isInternalToAdobeRuntime() ? ADOBE_STATE_STORE_ENDPOINT_PROD_INTERNAL : ADOBE_STATE_STORE_ENDPOINT_PROD,
+  [STAGE_ENV]: isInternalToAdobeRuntime() ? ADOBE_STATE_STORE_ENDPOINT_STAGE_INTERNAL : ADOBE_STATE_STORE_ENDPOINT_STAGE
```

Comment:
we have an internal issue for stage, the internal endpoint will not work but neither will the corp one in runtime. so let's keep it like this just writing this comment for visibility.